actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -419,14 +419,14 @@ def cookies
     end
 
   private
-    ContentTypeHeader = Struct.new :mime_type, :charset
-    NullContentTypeHeader = ContentTypeHeader.new nil, nil
+    ContentTypeHeader = Struct.new :mime_type, :extra, :charset
+    NullContentTypeHeader = ContentTypeHeader.new nil, nil, nil
 
     def parse_content_type(content_type)
       if content_type && match = CONTENT_TYPE_PARSER.match(content_type)
-        ContentTypeHeader.new(match[:type], match[:charset])
+        ContentTypeHeader.new(match[:type], match[:extra], match[:charset])
       else
-        NullContentTypeHeader
+        ContentTypeHeader.new(content_type, nil)
```

Comment:
Is this change required for GitHub?
Looks like this change is not tested.

Reply:
I don't think so.  I think we can remove the change (you're right, it seems not tested)